Sprint-3/alarmclock/alarmclock.js

```diff
@@ -1,16 +1,58 @@
-function setAlarm() {}
+var input_time = document.querySelector("#alarmSet");
+var title_time = document.querySelector("#titleTime");
+var timeRemaining = document.querySelector("#timeRemaining");
+var bUserStopped = false;
```

Comment:
You're using three different naming conventions for your variables here: camelCase, snake_case, and bHungarianNotation.

I'd recommend picking one naming convention to make things more consistent and easy to read. In JavaScript, most people use camelCase as the naming convention for variable names, so I'd recommend following that unless you have a reason to do something different.

Reply:
I changed them all to camelCases
thanks